lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_new_milestone_action.rb

```diff
@@ -49,19 +53,19 @@ def self.available_options
                                        env_name: 'GHHELPER_NEED_APPSTORE_SUBMISSION',
                                        description: 'True if the app needs to be submitted',
                                        optional: true,
-                                       is_string: false,
+                                       type: Boolean,
```

Comment:
Self-Review: As the `is_string` is deprecated on the ConfigItems, I replace them with the type of variable that we want to receive.